python-threatexchange/threatexchange/TE.py

```diff
@@ -13,35 +13,28 @@
 # General Python dependencies
 import os
 import re
-import requests
+import threading
```

Comment:
I see this import was added but I am missing where it was used.

Reply:
Whoops, leftover from a different attempt at fixing this (threadlocal).